prac_09/band.py

```diff
@@ -0,0 +1,35 @@
+"""
+CP1404 Practical
+Band class file
+"""
+
+
+class Band:
+    """Band class."""
+
+    def __init__(self, name=""):
+        """Construct a Band with an empty Musician list"""
+        self.name = name
+        self.musicians = []
+
+    def __str__(self):
+        """Return a string representation of a Band."""
+        return f"{self.name} ({self.musicians[0]}, {self.musicians[1]}, {self.musicians[2]}, {self.musicians[3]})"
```

Comment:
could be {self.name} {self.musicians} this would just print the name of the band and list of musicians